nexus/tests/integration_tests/scim.rs

```diff
@@ -204,3 +213,298 @@ async fn test_no_jit_for_saml_scim_silos(cptestctx: &ControlPlaneTestContext) {
     .await
     .expect("expected 401");
 }
+
+#[nexus_test]
+async fn test_scim_client_token_crud(cptestctx: &ControlPlaneTestContext) {
+    let client = &cptestctx.external_client;
+
+    // Create a Silo, then grant the PrivilegedUser the Admin role on it
+
+    const SILO_NAME: &str = "saml-scim-silo";
+    create_silo(&client, SILO_NAME, true, shared::SiloIdentityMode::SamlScim)
+        .await;
+
+    grant_iam(
+        client,
+        &format!("/v1/system/silos/{SILO_NAME}"),
+        shared::SiloRole::Admin,
+        USER_TEST_PRIVILEGED.id(),
+        AuthnMode::PrivilegedUser,
+    )
+    .await;
+
+    // Initially, there should be no tokens created during silo create.
+
+    let tokens: Vec<views::ScimClientBearerToken> =
+        object_get(client, &format!("/v1/system/scim/tokens?silo={SILO_NAME}"))
+            .await;
+
+    assert!(tokens.is_empty());
+
+    // Fleet admins can create SCIM client tokens
```

Comment:
Just going off of the comment here, but can we add a test that a non fleet admin cannot create a SCIM client token?